assets/src/pages/room/components/MediaControlButtons.tsx

```diff
@@ -105,10 +108,9 @@ const getAutomaticControls = (
       },
   {
     id: "chat",
-    icon: Chat,
+    icon: isMobile ? MenuDots : Chat,
     hover: isSidebarOpen ? "Close the chat" : "Open the chat",
```

Comment:
Should this be "Open the chat" or "Open the panel" (Maybe "Open the sidebar")?  🤔

Reply:
On the desktop dimension, it is implemented according to the design. I would change the label only on mobile to `Open the menu` because it is not a sidebar.